src/main/java/pl/tomaszdziurko/jvm_bloggers/utils/Validators.java

```diff
@@ -0,0 +1,14 @@
+package pl.tomaszdziurko.jvm_bloggers.utils;
+
+import org.apache.commons.validator.routines.UrlValidator;
+
+public class Validators {
+
+    private static final UrlValidator URL_VALIDATOR =
```

Comment:
\[Checkstyle\] WARNING: 'member def modifier' have incorrect indentation level 4, expected level should be 2\.